service/policy/db/subject_mappings.go

```diff
@@ -554,13 +549,8 @@ func selectMatchedSubjectMappingsSQL(subjectProperties []*policy.SubjectProperty
 		}
 
 		hasField := "each_condition->>'subject_external_selector_value' = '" + sp.GetExternalSelectorValue() + "'"
-		hasValue := "(each_condition->>'subject_external_values')::jsonb @> '[\"" + sp.GetExternalValue() + "\"]'::jsonb"
-		hasInOperator := "each_condition->>'operator' = 'SUBJECT_MAPPING_OPERATOR_ENUM_IN'"
-		hasNotInOperator := "each_condition->>'operator' = 'SUBJECT_MAPPING_OPERATOR_ENUM_NOT_IN'"
-		// Parses the json and matches the row if either of the following conditions are met:
-		where += "((" + hasField + " AND " + hasValue + " AND " + hasInOperator + ")" +
-			" OR " +
-			"(" + hasField + " AND NOT " + hasValue + " AND " + hasNotInOperator + "))"
+		// Parses the json and matches the row if the selector exists:
```

Comment:
I didn't touch this RPC to avoid merge conflicts, but perhaps you could remove this check and move it to the proto validation as part of this effort? https://github.com/opentdf/platform/pull/1658/files#diff-79178ab55805aac9e744330a45936ecde9e39baef7e8838764209f44b6a3f862R535 Similar to the work just done in GetAttributesByValueFQNs: https://github.com/opentdf/platform/pull/1657

Reply:
I also think we can ignore a check for values now that this route only drives based off selectors (explicitly _not_ values after your current work here): https://github.com/opentdf/platform/pull/1658/files#diff-79178ab55805aac9e744330a45936ecde9e39baef7e8838764209f44b6a3f862R542

Reply:
done